Snippets/Core/Core_6/Msmq/DLQOptInForTTBR.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Core6.Msmq
+{
+    using NServiceBus;
+
+    class DLQOptInForTTBR
+    {
+        DLQOptInForTTBR(EndpointConfiguration endpointConfiguration)
+        {
+            #region msmq-dlq-for-ttbr-optin
+
+            endpointConfiguration.UseTransport<MsmqTransport>()
+                .UseDeadLetterQueueForMessagesWithTimeToReachQueue();
```

Comment:
The MSMQ term is `TimeToReachQueue` https://msdn.microsoft.com/en-us/library/ms711440(v=vs.85).aspx